app/views/shared/activities/uploads/_upload_form.html.haml

```diff
@@ -7,7 +7,7 @@
         hint: { text: t("form.hint.activity.csv_file_recover_from_error_html", link: send(path_helper, instance, { type: type, format: :csv })) }
     - else
       = f.govuk_file_field :activity_csv,
-        label: { text: t("form.label.activity.csv_file"), hidden: !local_assigns[:show_hint] },
+        label: { text: t("form.label.activity.csv_file"), hidden: true },
```

Comment:
If it's always hidden, do we still need it?

Reply:
I've kept it in as it's still used by screen readers even when hidden. Maybe the context is explicit enough, though